db/users.js

```diff
@@ -21,8 +26,9 @@ const createUser = async (first_name, last_name, email, password) => {
         last_name: last_name,
         email: email,
         password: password,
+        settings: JSON.stringify(DEFAULT_SETTINGS),
       },
-      ["first_name", "last_name", "email", "password"]
+      ["first_name", "last_name", "email", "password", "settings"]
```

Comment:
I was thinking of making a settings page today to allow users to modify the theme and stuff you listed above, how would this "settings" attribute impact that?

Reply:
So I'll need to make a new route for updating the settings through a post request that will be the action of your form, but you have the user's settings coming into the EJS page through the 'user.settings' variable as shown on the home page, so you can set their default values to those.